### DISCUSSION TITLE
can i append something to folder views

### DISCUSSION THREAD

**Discussion by Sneaken** [microsoft/vscode-discussions#277] [Extension Development QnA] [answered]
I make heavy use of explorer.xxx elsewhere, 

so if I make a new view with a plugin, I won't be able to use other plugins,

How should I do this question？

![image](https://user-images.githubusercontent.com/37062838/204687364-5dea4d89-d796-4968-86ea-cc84d7308919.png)

**Comment by isidorn** [comment, marked as answer]
Extensions can not append UI inside the explorer. You can contribute another separate view.
Or your extension can contribute a FileSystemProvider to handle some types of files.
